.github/workflows/create_tag.yml

```diff
@@ -23,12 +23,23 @@ jobs:
       uses_custom_version_output: ${{ steps.tag_output.outputs.uses_custom_version }}
     env:
        SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL  }}
+       NEXUS_REPO_NPM: ${{vars.NEXUS_PUSH_REPOS_NPM}}
```

Comment:
Nowhere you need the env variable. Can we not use the `vars.NEXUS_PUSH_REPOS_NPM` directly, instead of setting one more env variable.?